packages/grant-explorer/src/features/discovery/CollectionCard.tsx

```diff
@@ -0,0 +1,49 @@
+import { ChainId } from "common";
+import {
+  Badge,
+  BasicCard,
+  CardContent,
+  CardHeader,
+  CardTitle,
+} from "../common/styles";
+import { CollectionBanner } from "./CardBanner";
+
+type CollectionCardProps = {
+  collection: {
+    id: string;
+    chainId: string;
```

Comment:
Let's use `ChainId` enum here directly instead of casting to it later. A great example of lifting the failure case up the tree.